src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -341,10 +341,14 @@ def load_arguments(self, _):
         c.argument('diskslist', diskslist_type)
         c.argument('restore_only_osdisk', arg_type=get_three_state_flag(), help='Use this flag to restore only OS disks of a backed up VM.')
         c.argument('restore_as_unmanaged_disks', arg_type=get_three_state_flag(), help='Use this flag to specify to restore as unmanaged disks')
-        c.argument('use_secondary_region', action='store_true', help='Use this flag to show recoverypoints in secondary region.')
-        c.argument('rehydration_duration', type=int, help='Set the maximum time, in days (between 10-30, both inclusive) for which the recovery point stays in hydrated state. Default: 15')
```

Comment:
Why we delete the prompt for the default value?

Reply:
![image](https://user-images.githubusercontent.com/26112239/127101844-8eaa331f-c9fc-40dd-9a0a-21dc3cdc4b57.png)

We have default value set in the function parameter. I think the framework already make use of it and creates a default help text. As it was showing duplicate texts to me. So i removed the text we added explicitly.

![image](https://user-images.githubusercontent.com/26112239/127102117-bcf2db10-8635-442f-99cc-ecd99ab3debd.png)

Reply:
OK, got it~